### PR TITLE
[New] Added test cases for record form nav

### PR DESCRIPTION
**Pull request by Srinivas9933**
Added test cases for record form nav component

- component.spec.js

### CODE REVIEW

File: app/javascript/components/record-form/nav/component.spec.js [hunk 1]
```diff
@@ -186,59 +179,39 @@ describe("<Nav />", () => {
...
+    expect(screen.getByTestId("Divider")).toBeInTheDocument();
   });
 
   it("renders a NavGroup component from record information and another one from the others forms groups />", () => {
-    expect(component.find(NavGroup)).to.have.lengthOf(2);
+    mountedComponent(<Nav {...props} />, initialState);
+    expect(screen.getByTestId("nav-list")).toBeInTheDocument();
   });
 
   it("renders the NavGroup component for record information open", () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/record-form/nav/component.spec.js:207]: Need something else to key off of

Comment (Srinivas9933) [reply app/javascript/components/record-form/nav/component.spec.js:207]: Done.

File: app/javascript/components/record-form/nav/component.spec.js [hunk 2]
```diff
@@ -186,59 +179,39 @@ describe("<Nav />", () => {
...
-    ].forEach(property => {
-      expect(navProps).to.have.property(property);
-      delete navProps[property];
-    });
-    expect(navProps).to.be.empty;
+    mountedComponent(<Nav {...props} />, initialState);
+    expect(screen.getByTestId("ConditionalWrapper")).toBeInTheDocument();
   });
 
   describe("when is a new record", () => {
```
Comment (jtoliver-quoin) [inline app/javascript/components/record-form/nav/component.spec.js:217]: From here on down, doesn't test what should be tested

Comment (Srinivas9933) [reply app/javascript/components/record-form/nav/component.spec.js:217]: Written test cases to check if there is a new record

**Pull request merged**